app/assets/stylesheets/components/_icon.scss

```diff
@@ -12,3 +12,22 @@
     background-image: url(image-path('ico-print.svg'));
   }
 }
+
+.ico-absolute {
+  background-repeat: no-repeat;
+  background-size: $h4;
```

Comment:
I wonder if there is a way to add some additional utility classes in order to allow for different sizes instead of just `$h4`? Unless you think this is pretty specific use case, in which case maybe this would make more sense being in bound to `h4` in `typography.css`?